src/main/java/bio/terra/service/profile/google/GoogleBillingService.java

```diff
@@ -43,7 +43,7 @@ private static CloudBillingClient accessClient(AuthenticatedUserRequest user) {
       // See https://developers.google.com/identity/protocols/application-default-credentials
       // for more information.
       List<String> scopes =
-          Collections.singletonList("https://www.googleapis.com/auth/cloud-platform");
+          Collections.singletonList("https://www.googleapis.com/auth/cloud-billing.readonly");
```

Comment:
```suggestion
          List.of("https://www.googleapis.com/auth/cloud-billing.readonly");
```
Also, since it's only used inside the `if`, it would be better to declare it inside the `if`.